WordPressMocks/src/main/assets/mocks/mappings/wpcom/sites/rest_v2_sites_158396482_gutenberg.json

```
@@ -0,0 +1,30 @@
+{
+  "request": {
+    "urlPattern": "/wpcom/v2/sites/158396482/gutenberg(/)?($|\\?.*)",
+    "method": "POST",
+    "headers": {
+      "Content-Type": {
+        "equalTo": "application/json; charset=UTF-8"
+      }
+    },
+    "bodyPatterns" : [ {
+      "equalToJson" : {
+        "editor" : "aztec",
+        "platform" : "mobile"
+      }
+    } ]
+  },
+  "response": {
+    "status": 200,
+    "jsonBody": {
+      "editor_mobile": "aztec",
+      "editor_web": "gutenberg",
+      "opt_in": "https:\/\/public-api.wordpress.com\/wpcom\/v2\/sites\/158396482\/gutenberg?editor=gutenberg&platform=mobile"
```

Updating this to point to the mocked API rather than `public-api.wordpress.com` would be ideal. Also, I don't think the escaping is needed here.

```suggestion
      "opt_in": "{{request.requestLine.baseUrl}}/wpcom/v2/sites/158396482/gutenberg?editor=gutenberg&platform=mobile"
```

---

Ahh you're right! I've gone ahead and commite, thanks!